### PR TITLE
fix: file restrictions

### PR DESCRIPTION
**Pull request by franciscog-kainos**
### JIRA link
https://companieshouse.atlassian.net/browse/LFA-1467

### Change description
Fixed bug identified during testing and made changes to accommodate for change in api's appeal get endpoint

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/modules/appeals-service/AppealsService.ts [hunk 1]
```diff
@@ -40,12 +40,13 @@ export class AppealsService {
 
         return await axios
             .post(uri, appeal, this.getConfig(token))
-            .then((response: AxiosResponse) => {
+            .then((response: AxiosResponse<string>) => {
                 if (response.status === CREATED && response.headers.location) {
                     loggerInstance()
                         .info(`${AppealsService.name} - save: created resource ${response.headers.location}`);
```
Comment (n-r-b) [inline src/modules/appeals-service/AppealsService.ts:46]: we could also add the response data to this, incase the API response changes

File: src/modules/appeals-service/AppealsService.ts [hunk 2]
```diff
@@ -40,12 +40,13 @@ export class AppealsService {
...
-                    return response.headers.location;
+                    return response.data.toString();
                 }
+                throw new Error('Could not create appeal resource');
```
Comment (n-r-b) [inline src/modules/appeals-service/AppealsService.ts:49]: Have you created a test for this?

File: test/modules/appeals-service/AppealsService.test.ts [hunk 3]
```diff
@@ -66,11 +67,11 @@ describe('AppealsService', () => {
...
                     }
                 )
-                .reply(201, {}, { 'location': RESOURCE_LOCATION });
+                .reply(201, appealId, { 'location': RESOURCE_LOCATION });
 
             await appealsService.save(appeal, BEARER_TOKEN)
-                .then((response) => {
-                    expect(response).to.equal(RESOURCE_LOCATION);
+                .then((response: string) => {
+                    expect(response).to.equal(appealId);
```
Comment (n-r-b) [inline test/modules/appeals-service/AppealsService.test.ts:74]: we can still expect the response to contain the location header?

File: src/modules/appeals-service/AppealsService.ts (hunk 2 above)
Comment (franciscog-kainos) [reply src/modules/appeals-service/AppealsService.ts:49]: I'll do that

File: test/modules/appeals-service/AppealsService.test.ts (hunk 3 above)
Comment (franciscog-kainos) [reply test/modules/appeals-service/AppealsService.test.ts:74]: the response is now the appealId not the header

Comment (franciscog-kainos) [reply test/modules/appeals-service/AppealsService.test.ts:74]: the response which is actually just the result of resolving the promise

**Pull request merged**